cmd/utils/flags.go

```diff
@@ -776,7 +776,6 @@ func setListenAddress(ctx *cli.Context, cfg *p2p.Config) {
 	}
 	if ctx.GlobalIsSet(ListenPort65Flag.Name) {
 		cfg.ListenAddr65 = fmt.Sprintf(":%d", ctx.GlobalInt(ListenPort65Flag.Name))
-		cfg.Eth65Enabled = true
```

Comment:
then don't need `IsSet` check and flag must have default value

Reply:
the logic is identical to eth66 🤷🏻‍♂️

Reply:
@vorot93 it's because 66 also has default value for config (not for cli flag). 
```
/Users/alex.sharov/projects/turbo-geth/node/defaults.go
```
probably you can add 65 default there